app/src/main/java/org/fossasia/openevent/app/common/data/models/Event.java

```diff
@@ -32,10 +44,18 @@
 @AllArgsConstructor
 @JsonNaming(PropertyNamingStrategy.KebabCaseStrategy.class)
 @Table(database = OrgaDatabase.class, allFields = true)
+@JsonInclude(JsonInclude.Include.NON_DEFAULT)
 @EqualsAndHashCode(exclude = { "eventDelegate", "analytics" })
 @SuppressWarnings({ "PMD.ExcessivePublicCount", "PMD.TooManyFields" })
+@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY, getterVisibility = JsonAutoDetect.Visibility.NONE,
+    setterVisibility = JsonAutoDetect.Visibility.NONE, creatorVisibility = JsonAutoDetect.Visibility.NONE,
+    isGetterVisibility = JsonAutoDetect.Visibility.NONE)
 public class Event implements Comparable<Event>, IHeaderProvider {
 
+    private static final String DEFAULT_STRING = "example";
+    private static final String DEFAULT_PROTOCOL = "http:";
+    private static final String DEFAULT_URL = "www.example.com";
+    private static final String DEFAULT_CURRENCY = "INR";
```

Comment:
See comment below about single responsibility

Reply:
Please read my comment below about setting default values. If needed, I'll initialize them in the presenter.